imports/ui/components/dataTable/helpers.jsx

```diff
@@ -33,5 +33,3 @@ export const transformColumn = (column) => {
   delete columnData.width;
   return columnData;
 };
-
-export const formatDate = (date) => moment(date).format('MM/DD/YYYY h:mm A');
```

Comment:
Maybe we should gracefully move code. e.g. by providing a link to a new function first.
With deprecation comment.

e.g. at the moment in my WIP PR I'm utilizing this function. So it will cause an issue as soon as I'll merge. (Not anymore since I noticed this change, but might have to)